networking_cisco/plugins/cisco/device_manager/plugging_drivers/aci_vlan_trunking_driver.py

```diff
@@ -237,14 +237,14 @@ def extend_hosting_port_info(self, context, port_db, hosting_device,
             # the actual segment ID comes from the confgi file
             if net.get('provider:network_type') == 'opflex':
                 if ext_dict.get('segmentation_id'):
-                    hosting_info['segmentation_id'] = (
-                        ext_dict['segmentation_id'])
+                    hosting_info['segmentation_id'] = ext_dict['segmentation_id']
             snat_subnets = self._core_plugin.get_subnets(
                 context.elevated(), {'name': [APIC_SNAT]})
             if snat_subnets:
-                hosting_info['snat_subnet_ids'] = []
+                hosting_info['snat_subnets'] = []
```

Comment:
There isn't a UT for this yet. I'll create a separate github issue for that, and will work on adding those.